scalafmt-tests/src/test/resources/scala3/Issues.stat

```diff
@@ -19,3 +19,21 @@ def f(s: String): Double =
   s match
     case re(v): String => v.toDouble
     case _             => 0.0
+<<< extends with scalafmt/issues/2479
+object Outer:
+  trait A
+
+  object Foo extends A
+  object Bar extends A
+
+  /** A doc comment */
```

Comment:
is this comment parsed as part of `object Bar` before?

Reply:
Yeah, that was the problem.

Reply:
hmm... should it be fixed in the parser and excluded from either tree (before or after)?